charm/charmcraft.yaml

```diff
@@ -75,6 +85,13 @@ config:
       description: |-
         SAML attribute used for the email.
       type: string
+    aws_endpoint_url:
```

Comment:
I think the recommended style for naming charm configurations is kebab case: https://discourse.charmhub.io/t/standard-format-for-common-charm-files/7438. Since this is a new charm, maybe we should use kebab case instead?

Reply:
The current library used for Django 12 factor uses underscores, and changing it would result in env variables like `DJANGO_SAML-USERNAME`. 

I have created a new story (ISD-1684) to do that among other things when integrating with the last Django 12 factor library. If you agree, I will tackle it there. WDYT?

Reply:
Understand, I think the latest version of `xiilib` has already fix this issue, so I will just migrate all the configurations once the 12-factor toolchain has been upgraded, thanks!